util/build_compilation_database.sh

```diff
@@ -8,7 +8,7 @@ TMP_DIR=`mktemp -d`
 echo ${TMP_DIR}
 AWS_LC_BUILD="${TMP_DIR}/AWS-LC-BUILD"
 
-MY_CMAKE_FLAGS=("-GNinja" "-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_EXPORT_COMPILE_COMMANDS=ON")
+MY_CMAKE_FLAGS=("-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_EXPORT_COMPILE_COMMANDS=ON" "-DDISABLE_GO=ON")
```

Comment:
Curious does clang-tidy not work with ninja and our go build?

Reply:
clang-tidy gets run inside of a docker image, and the docker image does not have Ninja installed.